### ISSUE TITLE
"Input.use_accumulated_input = false" not working

### ISSUE THREAD

**Issue by shanayyy** [godotengine/godot#93769]
### Tested versions

Reproducible: v4.3.beta2.official [b75f0485b]

### System information

Godot v4.3.beta2 - macOS 14.5.0 - Vulkan (Forward+) - integrated Apple M2 Pro - Apple M2 Pro (10 Threads)

### Issue description

Disables input accumulation, but input events are still called together every frame.

### Steps to reproduce

Set the maximum fps to 1 and print the time in the input callback. When the key is pressed continuously, the printed time is repeated within each frame. This prevents getting the exact time of the input in Godot.

### Minimal reproduction project (MRP)

[new-game-project.zip](https://github.com/user-attachments/files/16043456/new-game-project.zip)

**Comment by precup** [comment]
This doesn't really feel like a case that `use_accumulated_input` is intended to cover, but I can also see why it would be counterintuitive or unwanted. Needing to generate the repeated key presses at the appropriate times seems like a difficult scheduling problem since there aren't real events occurring to trigger them.

That said, I can confirm this is happening and need testing can be removed.

**Comment by precup** [comment]
@shanayyy Would your use case be fixed by https://github.com/godotengine/godot-proposals/issues/552? Requiring the event handlers to be called at the precise times the events occurred seems like a much stronger condition than just informing you of when it was.

**Comment by shanayyy** [comment]
If I understand correctly, does this mean that even if my fps is 1, the input event callback will be called immediately when the player presses a key?  This would definitely be the better solution 😁

**Comment by precup** [comment]
No, they would still be called at the same time they are now, but they would include a timestamp in the input event that could be used in place of `Time.get_ticks_msec()`, so each would have a different timestamp.
